tests/e2e/api/transaction_submission.test.ts

```diff
@@ -88,12 +88,12 @@ describe("transaction submission", () => {
       });
       expect(rawTxn.rawTransaction instanceof Uint8Array).toBeTruthy();
       const deserializer = new Deserializer(rawTxn.rawTransaction);
-      const desirializedTransaction = RawTransaction.deserialize(deserializer);
-      expect(desirializedTransaction instanceof RawTransaction).toBeTruthy();
-      expect(desirializedTransaction.payload instanceof TransactionPayloadEntryFunction).toBeTruthy();
+      const deserializedTransaction = RawTransaction.deserialize(deserializer);
+      expect(deserializedTransaction instanceof RawTransaction).toBeTruthy();
+      expect(deserializedTransaction.payload instanceof TransactionPayloadEntryFunction).toBeTruthy();
```

Comment:
We can either fix all these misspellings now (this is copied a lot of places), or come back on a pass Friday, where we cleanup all mispelling.